### ISSUE TITLE
Customer-Vendor-Admin Support System

### ISSUE THREAD

**Issue by Tangol** [wcmarketplace/dc-woocommerce-multi-vendor#683]
WCMp is missing a way for the Vendor to directly contact the Admin regarding issues with payments, orders, Knowledge Base, bugs etc.
This system also works hand in hand with https://github.com/wcmarketplace/dc-woocommerce-multi-vendor/issues/682, as the vendor will need to be able to contact the admin, and the admin needs to be able to keep track of this communication.

This needs to be extended to the customer area also, as the customer will want to contact the vendor for refunds, returns and other inquiries. 

I've found a nice [module](https://codecanyon.net/item/woocommerce-support-ticket-system/17930050) that enables customer-admin communication.
If it's too hard for you guys to develop it from scratch, maybe you can add support for the above module.

**Comment by moumitahalder** [comment]
@Tangol, yes we do have plan to integrate this module. Hopefully, within next few month, we can add this.
